upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -418,6 +421,34 @@ func (c *ApplyClusterCmd) Run() error {
 	l.WorkDir = c.OutDir
 	l.ModelStore = modelStore
 
+	iamLifecycle := lifecyclePointer(fi.LifecycleSync)
+	networkLifecycle := lifecyclePointer(fi.LifecycleSync)
+	clusterLifecycle := lifecyclePointer(fi.LifecycleSync)
+
+	switch c.Phase {
```

Comment:
Can you explain why?  In particular, why not just run two commands?

A goal here is to eventually make networks a top level object, like Cluster.